README.md

```diff
@@ -232,7 +218,7 @@ network stack. This is done in the following steps:
 Modify the command that runs the Image Caption Generator REST endpoint to map an additional port in the container to a
 port on the host machine. In the example below it is mapped to port `8088` on the host but other ports can also be used.
 
-    docker run -it -p 5000:5000 -p 8088:8088 --name max-im2txt max-im2txt
+    docker run -it -p 5000:5000 -p 8088:8088 --name max-im2txt codait/max-image-caption-generator
```

Comment:
While we are changing this, I wonder if the `--name` should also change? im2txt was more a WIP name before publishing, so perhaps we should change that to `max-image-caption-generator` (even though a bit more verbose)

Reply:
I would agree here and I can update this for the code pattern, but this is how all the model repos have their naming, based on the original model name. We originally just copied this from the model repo, so should we update all the model repos as well? I could do it while adding the docker hub steps when I open those prs